### ISSUE TITLE
Is there a way to validate against multiple fields in one IsValidField func?

### ISSUE THREAD

**Issue by collinalexbell** [asaskevich/govalidator#273]
Lets say I have a simple struct with a min and max field and I want to make sure that min < max. Does govalidator provide a means to do this?

**Comment by rockertux** [comment]
Yes. Last example in this section of the docs: https://github.com/asaskevich/govalidator#custom-validation-functions
